artifacts

```diff
@@ -57,6 +57,20 @@ upload() {
 
 }
 
+get() {
+    if [ "${INPUT_WORKFLOW_NAME}" = "default" ]; then
+      echo "Set a workflow name to get his Artifacts name"
+      exit 2
+    fi
+
+    NAME="github:${REPOSITORY}:staging-${COMMIT_TIMESTAMP}.${COMMIT_SHORT_REVISION}.${INPUT_WORKFLOW_NAME}.${RUN_NUMBER}"
+    LAST_SUCCESS_LINK=$(curl -u ${USER}:${PASSWORD} --silent --fail --show-error -L -I "${URL}/last_success/${NAME}" | grep -E "^location:.*\/download\/([^\/]+)\/");
+    GET_NAME=$(echo "${LAST_SUCCESS_LINK}" | sed -E 's/^location:.*\/download\/([^\/]+)\//\1prolonged\1/');
```

Comment:
Verify this value as it seems to have twice the name of the artifacts in the output.